src/geo/ui/legends/bubble/legend-template.tpl

```diff
@@ -1,18 +1,32 @@
 <div class="Bubble-container u-flex u-justifySpace">
   <ul class="Bubble-numbers u-flex u-justifySpace">
-    <% for (var i = 0; i<labels.length; i++) { %>
-      <li class="Bubble-numbersItem CDB-Text CDB-Size-small" style="bottom: <%- labelPositions[i] %>%"><%= formatter.formatNumber(labels[i]) %></li>
+    <% if (hasCustomLabels) { %>
+      <li class="Bubble-numbersItem CDB-Text CDB-Size-small" style="bottom: 0%"><%- labels[0] %></li>
```

Comment:
I guess we could set labelPositions to `[ 0, 100 ]` when custom labels are applied and remove this conditional.